runtime/src/lib.rs

```diff
@@ -115,7 +75,7 @@ fn wait_for_cmd(_mbox: &mut Mailbox) {
     //}
 }
 
-fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
+fn handle_and_respond(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
```

Comment:
Maybe add a comment to this function the explain what handle_and_respond means. Maybe something like

```rust
/// Handles any pending mailbox command and writes response to the mailbox
```